man/namedlist.Rd

```diff
@@ -0,0 +1,12 @@
+% Generated by roxygen2: do not edit by hand
```

Comment:
Because I was dump and didn't realize that I can have @importFrom without the docs :-)